docs/user-guide/configuration-files/strongbox-yaml.md

```diff
@@ -0,0 +1,398 @@
+# General
+
+This resource file describes the configuration of the server.
+
+The property controlling this file is `strongbox.config.file`. The default location for this resource is `etc/conf/strongbox.yaml`.
+
+The cases in which this file is required are:
+* During the server launch
+* For a configuration import from another server
+* For a configuration export of the server
+
+# The Configuration File
+
+The following is an example of a `strongbox.yaml` configuration file:
+
+    configuration:
+      instanceName: strongbox
+      version: ${project.version}
+      revision: ${strongbox.revision}
+      baseUrl: http://localhost:48080/
+      port: 48080
+      sessionConfiguration:
+        timeoutSeconds: 3600
+      remoteRepositoriesConfiguration:
+        checkIntervalSeconds: 60
+        heartbeatThreadsNumber: 5
+        retryArtifactDownloadConfiguration:
+          timeoutSeconds: 60
+          maxNumberOfAttempts: 5
+          minAttemptsIntervalSeconds: 5
+      corsConfiguration:
+        allowedCredentials: true
+        maxAge: 600
+        allowedOrigins:
+          - "*"
+        allowedMethods:
+          - GET
+          - PUT
+          - POST
+          - DELETE
+          - OPTIONS
+        allowedHeaders:
+          - Accept
+          - Accepts
+          - Authorization
+          - Access-Control-Allow-Headers
+          - Access-Control-Request-Headers
+          - Access-Control-Request-Method
+          - DNT
+          - Keep-Alive
+          - User-Agent
+          - X-Requested-With
+          - If-Modified-Since
+          - Cache-Control
+          - Content-Type
+          - Content-Range,Range
```

Comment:
@ptirador / @fuss86 shouldn't this be `- Content-Range` and `- Range` on a new line?

And another question - will this format work as well?

```yaml
allowedHeaders: ["Accept", "Accepts", "Authorization", ...]
```

Reply:
> And another question - will this format work as well?

I don't think so.

> YAML's block collections use indentation for scope and begin each member on its own line. Block sequences indicate each member with a dash (“-”). 

from

https://yaml.org/spec/1.0/#id2489726